.github/workflows/live-deploy.yml

```diff
@@ -12,18 +14,26 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Run Typescript Tests
+      - name: Setup Node
+        uses: actions/setup-node@v2.1.0
+        with:
+          node-version: 10.x
+
+      - name: Build Web Runtime
         run: |
+          npm install
           tools/sigh webpack
-          tools/sigh test --all
+
+      - name: Run Typescript Tests
```

Comment:
I might have asked earlier, but do the steps happen in sequence?

Reply:
Yes, these happen in sequence.